src/main/java/io/spring/releasenotes/generator/ReleaseNotesSection.java

```diff
@@ -44,7 +45,6 @@
 
 	ReleaseNotesSection(String title, String emoji, List<String> labels) {
 		Assert.hasText(title, "Title must not be empty");
-		Assert.hasText(emoji, "Emoji must not be empty");
```

Comment:
This feels a bit odd to have a nullable field in between two mandatory fields. If we were to merge this, the `emoji` parameter should probably move.